samples/python/visual_language_chat/visual_language_chat.py

```diff
@@ -36,7 +36,7 @@ def read_image(path: str) -> Tensor:
 
     '''
     pic = Image.open(path).convert("RGB")
-    image_data = np.array(pic.getdata()).reshape(1, 3, pic.size[1], pic.size[0]).astype(np.byte)
+    image_data = np.array(pic.getdata()).reshape(1, pic.size[1], pic.size[0], 3).astype(np.byte)
     return Tensor(image_data)
```

Comment:
@akuporos do you think OpenVINIO Python can support ov.Tensor creation from Pillow image directly ?

Reply:
yes, we can do it

Reply:
ticket #156053